src/modules/Control/Control.scss

```diff
@@ -0,0 +1,25 @@
+.j-control {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  background-repeat: no-repeat;
+
+  &--type {
+
+    &-edit {
```

Comment:
also maybe lets have a map with all the types and images path and lets this be generated through that.